test/functional/apps/settings/_index_pattern_results_sort.js

```diff
@@ -0,0 +1,136 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var SettingsPage = require('../../../support/pages/SettingsPage');
+  var expect = require('intern/dojo/node!expect.js');
+  var Promise = require('bluebird');
+
+  return function (bdd) {
+    bdd.describe('index result field sort', function describeIndexTests() {
+      var common;
+      var settingsPage;
+      var remote;
+
+      bdd.before(function () {
+        common = new Common(this.remote);
+        settingsPage = new SettingsPage(this.remote);
+        remote = this.remote;
+      });
+
+      bdd.beforeEach(function be() {
+        return settingsPage.createIndex();
```

Comment:
This beforeEach seems to be problematic. I'm getting the following error intermittently:

```
! timeout
Error: timeout
  at attempt  </Users/matt/Code/kibana/test/support/pages/Common.js:25:16>
  at tryCatcher  <node_modules/bluebird/js/browser/bluebird.js:4559:23>
  at Promise._settlePromiseFromHandler  <node_modules/bluebird/js/browser/bluebird.js:2687:31>
  at Promise._settlePromiseAt  <node_modules/bluebird/js/browser/bluebird.js:2761:18>
  at Promise._settlePromises  <node_modules/bluebird/js/browser/bluebird.js:2877:14>
  at Async._drainQueue  <node_modules/bluebird/js/browser/bluebird.js:175:16>
  at Async._drainQueues  <node_modules/bluebird/js/browser/bluebird.js:185:10>
  at Immediate.Async.drainQueues [as _onImmediate]  <node_modules/bluebird/js/browser/bluebird.js:67:14>
  at processImmediate [as _immediateCallback]  <timers.js:367:17>
! timeout
Error: timeout
  at attempt  </Users/matt/Code/kibana/test/support/pages/Common.js:25:16>
  at tryCatcher  <node_modules/bluebird/js/browser/bluebird.js:4559:23>
  at Promise._settlePromiseFromHandler  <node_modules/bluebird/js/browser/bluebird.js:2687:31>
  at Promise._settlePromiseAt  <node_modules/bluebird/js/browser/bluebird.js:2761:18>
  at Promise._settlePromises  <node_modules/bluebird/js/browser/bluebird.js:2877:14>
  at Async._drainQueue  <node_modules/bluebird/js/browser/bluebird.js:175:16>
  at Async._drainQueues  <node_modules/bluebird/js/browser/bluebird.js:185:10>
  at Immediate.Async.drainQueues [as _onImmediate]  <node_modules/bluebird/js/browser/bluebird.js:67:14>
  at processImmediate [as _immediateCallback]  <timers.js:367:17>
  at Command.then  <node_modules/intern/node_modules/leadfoot/Command.js:542:10>
  at SettingsPage.createIndex  </Users/matt/Code/kibana/test/support/pages/SettingsPage.js:235:7>
  at Suite.be  </Users/matt/Code/kibana/test/functional/apps/settings/_index_pattern_results_sort.js:20:28>
  at Suite.target.(anonymous function).dispatcher [as beforeEach]  <node_modules/intern/node_modules/dojo/aspect.ts:102:35>
  at <node_modules/intern/lib/Suite.js:263:43>
  at new Promise  <node_modules/intern/node_modules/dojo/Promise.ts:411:3>
  at runWithCatch  <node_modules/intern/lib/Suite.js:262:15>
  at next  <node_modules/intern/lib/Suite.js:267:17>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/node_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/node_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at process._tickCallback  <node.js:355:11>
```